app/__tests__/home.test.ts

```diff
@@ -1,26 +1,7 @@
-import { UITestBuilder } from '@antiwork/shortest';
+import { test } from '@antiwork/shortest';
```

Comment:
We have this in the README, not sure why this doesn't match.

Reply:
Oh! I updated the the README with the correct syntax.